### PR TITLE
feat: setup a GAR repository for CFT images

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

File: infra/terraform/test-org/ci-project/gar.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module "artifact_registry" {
+  source  = "GoogleCloudPlatform/artifact-registry/google"
+  version = "~> 0.1"
+
+  location      = "us"
+  project_id    = local.project_id
+  format        = "DOCKER"
+  repository_id = "gcr.io"
```
Comment (bharathkkb) [inline infra/terraform/test-org/ci-project/gar.tf:8]: Is this naming for migration?

**Review by bharathkkb — APPROVED** [review]
Thanks @g-awmalik!

File: infra/terraform/test-org/ci-project/gar.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module "artifact_registry" {
+  source  = "GoogleCloudPlatform/artifact-registry/google"
+  version = "~> 0.1"
```
Comment (apeabody) [inline infra/terraform/test-org/ci-project/gar.tf:3]: nit: any reason not to start with?
```suggestion
  version = "~> 0.2"
```

**Comment by g-awmalik** [comment]
Applied.

**Pull request merged**